script/intentfest/conversation.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+"""Simulate a conversation to check what the conversation would look like."""
+from __future__ import annotations
+
+import argparse
+import traceback
+
+from .const import LANGUAGES
+from .sentenceParser import SentenceParser
+from .util import get_base_arg_parser
+
+
+def get_arguments() -> argparse.Namespace:
+    """Get parsed parameters passed in arguments."""
+    parser = get_base_arg_parser()
+    parser.add_argument(
+        "--language",
+        required=True,
+        type=str,
+        choices=LANGUAGES,
+        help="The language to validate.",
+    )
+    parser.add_argument(
+        "--debug-response",
+        help="Set to 1 to show more info about response.",
+    )
+
+    return parser.parse_args()
+
+
+class InstructionsI18n:
+    def __init__(self, languageKey="en") -> None:
+        self.fallbackLanguage = "en"
+
+        # Feel free to add texts in your language ;)
+        self.instructionsI18n = {
+            "en": {
```

Comment:
translations should not live inside the python code. create a new folder under intentfest.

Reply:
but also, we can just keep it in English for now? Otherwise we need to add validation for intentfest translations etc …

Reply:
OK! 😄 Thanks